app/main.py

```diff
@@ -99,27 +117,29 @@ def perform_topup(self) -> None:
         )
 
         if current_data_limit <= 0:
-            print(f'Current data limit could not be retrieved, will retry.')
+            print(f'Current data limit could not be retrieved, will retry.', file=sys.stderr)
             return
 
-        if balance_in_gb >= float(lifeguard.top_up_threshold_gb):
+        if balance_in_gb >= lifeguard.top_up_threshold_gb:
             print(
                 'You still have enough data: '
                 f'Data remaining {balance_in_gb}, '
                 f'Threshold {lifeguard.top_up_threshold_gb}'
             )
-        elif current_data_limit >= float(lifeguard.max_gb):
+        elif current_data_limit >= lifeguard.max_gb:
             print(
                 "You've exceeded your maximum quota: "
                 f'Current data limit {current_data_limit}, '
                 f'Max data limit {lifeguard.max_gb}'
             )
         else:
-            if bool(lifeguard.dryrun) is not False:
+            if configured_bool(lifeguard.dryrun) is True:
```

Comment:
Is this cast still needed?

Reply:
oh no forgot that I handle it in `load_config` now -- my bad 😑